spec/predicates/simple-verification.md

```diff
@@ -0,0 +1,108 @@
+# Predicate type: Simple Verification
+
+Type URI: https://in-toto.io/attestation/simple-verification/v0.1
+
+Version: 0.1
+
+Authors: Tom Hennen (@TomHennen), Andrew McNamara (@arewm)
+
+## Purpose
+
+A Simple Verification Attestation (SVA) communicates that an artifact has been verified against one or more policies, and records the properties that were verified. This enables consumers to trust a concise summary of verification results, without requiring access to all underlying attestations or policies.
+
+## Use Cases
+
+-   Allowing software consumers to delegate complex policy decisions to a trusted verifier, and rely on a simple summary of which properties were verified.
+-   Supporting evolving verification needs (e.g., SLSA, vulnerability scanning, organizational policies) without coupling the SVA format to any specific framework.
+-   Enabling verifiers to add additional information (e.g., policy references, expiration, property-level timing) as needed, while maintaining a simple core schema.
+
+## Prerequisites
+
+This predicate depends on the [in-toto Attestation Framework]. Familiarity with the concept of attestation statements and the use of URIs for identifying verifiers and resources is assumed.
+
+## Model
+
+An SVA is an attestation that a verifier (`verifier.id`) has checked an artifact (`resourceUri`) at a specific time (`timeVerified`) and asserts a set of verified properties (`properties`). The SVA does not include information for reproducing the verification result. Instead, verifiers MAY generate additional provenance attestations for the simple verification attestation as needed.
+
+## Schema
+
+```jsonc
+{
+  // Standard attestation fields:
+  "_type": "https://in-toto.io/Statement/v1",
+  "subject": [{ ... }],
+
+  // Predicate:
+  "predicateType": "https://in-toto.io/attestation/simple-verification/v0.1",
+  "predicate": {
+    "verifier": {
+      "id": "<URI>"
+    },
+    "timeVerified": "<TIMESTAMP>",
+    "resourceUri": "<artifact-URI>",
+    "properties": ["<SCOPED_VERIFIED_PROPERTY>", ...]
+  }
+}
+```
+
+### Parsing Rules
+
+This predicate follows the
+[in-toto Attestation Framework's parsing rules](../v1/README.md#parsing-rules).
+
+### Fields
+
+**`verifier`, required** object
+
+> Identifies the entity that performed the verification.
+
+**`verifier.id`, required** string (TypeURI)
+
+> URI indicating the verifier's identity.
+
+**`timeVerified`, required** string (Timestamp)
+
+> Timestamp indicating what time the verification occurred.
+
+**`resourceUri`, required** string (ResourceURI)
+
+> URI that identifies the resource associated with the artifact being verified.
+
+**`properties`, required** array of string
+
+> Indicates the properties verified for the artifact.
```

Comment:
Do you have any prescriptive guidance on how to name properties, or is this entirely up to the SVA producer?

Reply:
I added some further guidance here

```
**`properties`, required** array of string

> Indicates the properties verified for the artifact. These SHOULD be scoped according
> to the framework being verified or the verifier's policy rules.
```